From 3a50c4b196c081e3375ab2f0183283e4c74b270b Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Thu, 2 Mar 2017 21:41:49 +0000
Subject: [PATCH 2407/2748] HADOOP-14138. Remove S3A ref from META-INF service
 discovery, rely on existing core-default entry.
 Contributed by Steve Loughran

(cherry picked from commit 51fefbe38bf40373d2855163fa6444821b0f994c)

Change-Id: I265f8638a7653761c3852da62e6efef7d43a1104
---
 .../services/org.apache.hadoop.fs.FileSystem       |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/src/main/resources/META-INF/services/org.apache.hadoop.fs.FileSystem b/hadoop-tools/hadoop-aws/src/main/resources/META-INF/services/org.apache.hadoop.fs.FileSystem
index 0e3c42a..3cd1d6b 100644
--- a/hadoop-tools/hadoop-aws/src/main/resources/META-INF/services/org.apache.hadoop.fs.FileSystem
+++ b/hadoop-tools/hadoop-aws/src/main/resources/META-INF/services/org.apache.hadoop.fs.FileSystem
@@ -15,4 +15,3 @@
 
 org.apache.hadoop.fs.s3.S3FileSystem
 org.apache.hadoop.fs.s3native.NativeS3FileSystem
-org.apache.hadoop.fs.s3a.S3AFileSystem
-- 
1.7.9.5

